plutus-use-cases/src/Language/PlutusTx/Coordination/Contracts/Vesting.hs

```diff
@@ -1,45 +1,59 @@
--- | Vesting scheme as a PLC contract
+{-# LANGUAGE TypeFamilies      #-}
+{-# LANGUAGE ConstraintKinds   #-}
+{-# LANGUAGE TypeOperators     #-}
 {-# LANGUAGE DataKinds         #-}
 {-# LANGUAGE DeriveGeneric     #-}
 {-# LANGUAGE FlexibleContexts  #-}
+{-# LANGUAGE MonoLocalBinds    #-}
+{-# LANGUAGE NoImplicitPrelude #-}
 {-# LANGUAGE OverloadedStrings #-}
-{-# LANGUAGE RecordWildCards   #-}
+{-# LANGUAGE NamedFieldPuns    #-}
 {-# LANGUAGE TemplateHaskell   #-}
 {-# LANGUAGE TypeApplications  #-}
 {-# LANGUAGE NoImplicitPrelude #-}
 {-# OPTIONS_GHC -fno-ignore-interface-pragmas #-}
 {-# OPTIONS_GHC -fno-specialise #-}
+-- $vesting
```

Comment:
Maybe haddock is choking because this isn't a Haddock comment? I don't really know how these `$` references work.

Reply:
It was tripping earlier that's why I put the `$` in in the first place, I can't reproduce it on my machine though :(

Reply:
https://github.com/haskell/haddock/issues/965